en/guide/migrating-5.md

```diff
@@ -95,44 +127,186 @@ This should not affect your code if you follow the Express 4 documentation of [a
 
+```js
+// v4
+app.get('/user', (req, res) => {
+  res.redirect('/users', 301)
+})
+
+// v5
+app.get('/user', (req, res) => {
+  res.redirect(301, '/users')
+})
+```
+
 
 <h4 id="magic-redirect">res.redirect('back') and res.location('back')</h4>
 
 Express 5 no longer supports the magic string `back` in the `res.redirect()` and `res.location()` methods. Instead, use the `req.get('Referrer') || '/'` value to redirect back to the previous page. In Express 4, the res.`redirect('back')` and `res.location('back')` methods were deprecated.
 
+```js
+// v4
+app.get('/user', (req, res) => {
+  res.redirect('back')
+})
+
+// v5
+app.get('/user', (req, res) => {
+  res.redirect(req.get('Referrer') || '/')
+})
+```
+
 <h4 id="res.send.body">res.send(body, status)</h4>
 
 Express 5 no longer supports the signature `res.send(obj, status)`. Instead, set the status and then chain it to the `res.send()` method like this: `res.status(status).send(obj)`.
 
+```js
+// v4
+app.get('/user', (req, res) => {
+  res.send({ name: 'Ruben' }, 200)
+})
+
+// v5
+app.get('/user', (req, res) => {
+  res.status(200).send({ name: 'Ruben' })
+})
+```
+
 <h4 id="res.send.status">res.send(status)</h4>
 
 Express 5 no longer supports the signature `res.send(status)`, where `status` is a number. Instead, use the `res.sendStatus(statusCode)` function, which sets the HTTP response header status code and sends the text version of the code: "Not Found", "Internal Server Error", and so on.
 If you need to send a number by using the `res.send()` function, quote the number to convert it to a string, so that Express does not interpret it as an attempt to use the unsupported old signature.
 
+```js
+// v4
+app.get('/user', (req, res) => {
+  res.send(200)
+})
+
+// v5
+app.get('/user', (req, res) => {
+  res.sendStatus(200)
+})
+```
+
 <h4 id="res.sendfile">res.sendfile()</h4>
 
 The `res.sendfile()` function has been replaced by a camel-cased version `res.sendFile()` in Express 5.
 
+```js
+// v4
+app.get('/user', (req, res) => {
+  res.sendfile('/path/to/file')
+})
+
+// v5
+app.get('/user', (req, res) => {
+  res.sendFile('/path/to/file')
+})
+```
+
 <h3>Changed</h3>
 
 <h4 id="path-syntax">Path route matching syntax</h4>
 
 Path route matching syntax is when a string is supplied as the first parameter to the `app.all()`, `app.use()`, `app.METHOD()`, `router.all()`, `router.METHOD()`, and `router.use()` APIs. The following changes have been made to how the path string is matched to an incoming request:
 
 - The wildcard `*` must have a name, matching the behavior of parameters `:`, use `/*splat` instead of `/*`
-- The optional character `?` is no longer supported, use braces instead: `/:file{.:ext}`.
+
+```js
+// v4
+app.get('/*', async (req, res) => {
+  res.send('ok')
+})
+
+// v5
+app.get('/*splat', async (req, res) => {
+  res.send('ok')
+})
+```
+
+{% capture note_wildcard %}
+If your defined route is `/*splat`, it will not include `/`, so you will need to make the route optional.
```

Comment:
@blakeembrey 
Is this wording correct? It’s a recurring problem i’ve seen, which is why I wanted to add that note, but I’m not sure if this is the best phrasing.